test/test_charm.py

```diff
@@ -323,21 +315,21 @@ def _on_stor4_attach(self, event: ops.StorageAttachedEvent):
             fi
             """,
         )
-        fake_script(
-            self,
+        fake_script_fixture.fake_script(
             "storage-list",
             """
             echo '["disks/0"]'
             """,
         )
 
-        assert self.meta.storages['stor1'].multiple_range is None
-        assert self.meta.storages['stor2'].multiple_range == (2, 2)
-        assert self.meta.storages['stor3'].multiple_range == (2, None)
-        assert self.meta.storages['stor-4'].multiple_range == (2, 4)
-        assert self.meta.storages['stor-plus'].multiple_range == (10, None)
+        meta = request.node.get_closest_marker("charm_meta").args[0]  # type: ignore
```

Comment:
Yeah, I think this is pretty icky -- taking what was a simple attribute/variable and routing it indirectly via pytest "markers" by name (and we have to add it to the `pyproject.toml` even)...